### PR TITLE
Add mailjet.com

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Mailjet is an email marketing and transactional email service platform that helps businesses design, send, and track email campaigns efficiently. The platform provides 2FA for the account login verification process. The website is ranked 52,203 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=mailjet.com). The website provides documentation on how to turn on 2FA. And Sorry for that I have tired my best but can't find the SVG logo. ❤️

### CODE REVIEW

File: entries/m/mailjet.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Mailjet": {
+    "domain": "mailjet.com",
+    "img": "mailjet.com.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://documentation.mailjet.com/hc/en-us/articles/10553745110043-Two-factor-authentication-2FA",
```
Comment (Carlgo11) [inline entries/m/mailjet.com.json]:
```suggestion
    "documentation": "https://documentation.mailjet.com/hc/articles/10553745110043",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
It looks like you included other commits into this PR. Please remove commit 9ed4991543d9e1436465db5622271cfd33f9f46b. Advise if you need help to do it.

Here's their SVG icon:
icon

File: entries/m/mailjet.com.json (hunk 1 above)
Comment (Yang-Nankai) [reply entries/m/mailjet.com.json]: Thank you! I have fixed this problem by myself! ❤️

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
Great job! 👏 
Your contribution will be visible in a little while

**Pull request merged**